controllers/metrics/metrics.go

```diff
@@ -26,10 +26,18 @@ var (
 		Name:      "requests",
 		Help:      "requests against the grafana api per instance",
 	}, []string{"instance_name", "path", "method", "status"})
+
+	DashboardUrlRequests = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
Example of such metric: `grafana_operator_dashboards_requests{dashboard="grafanadashboard-from-url",method="GET",path="/integr8ly/grafana-operator/master/deploy/examples/remote/grafana-dashboard.json",status="200"} 2`
As you can see, there's no reference for a namespace, so I wonder what would happen when you have dashboards with the same name in multiple namespaces?

Reply:
Good point. I wonder, do we even need to include the dashboard? The metric is meant to count the requests against the dashboard API overall, it doesn't have to be per dashboard.

Reply:
I think we're talking about different metrics here. My point was only about counting the number of requests for dashboards pulled from URLs. :)

Reply:
Ah I see. This metric does count the number of requests against the Grafana dashboard API. I've now added the namespace to the dashboard name label.